app/src/main/java/mobile/programming/musicapp/service/InstrumentService.kt

```diff
@@ -11,21 +11,17 @@ import retrofit2.Response
 class InstrumentService {
     internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
         val _instruments = MutableLiveData<ArrayList<Instrument>>()
-        val service = RetrofitClientInstance.retrofitInstance?.create(
-            MusicAppDAO::class.java)
+        val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
-        call?.enqueue(object : Callback<ArrayList<Instrument>> {
 
-            override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {
-                //log here
-            }
+        call?.enqueue(object : Callback<ArrayList<Instrument>> {
+            override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) { /*...*/ }
```

Comment:
Cleaned up this empty function and commented the empty section in it as recommended in [https://kotlinlang.org/docs/reference/coding-conventions.html#colon](url) to indicate easier that it is empty.

Reply:
Since you're in an onFailure call, it means that something went wrong.  A good way to improve the codebase and reduce technical debt is to add a proper logging statement here.  As an empty function, it is currently taking very good information about what went wrong, and throwing it away.  Developers will want that information when something goes wrong; otherwise, it will be a mysterious crash!